doc/sphinx/source/using.rst

```diff
@@ -208,6 +208,10 @@ pocl.
  POCL_TTASIM0_PARAMETERS will be passed to the first ttasim driver instantiated
  and POCL_TTASIM1_PARAMETERS to the second one.
 
+- **POCL_LLVM_VERIFY** environment variable can be set to ``0``
```

Comment:
Alphabetical order broken.

Reply:
Clarify that it verifies the linked input only? Would be useful to verify parallel.bc.